src/features/device-logs/lib/backends/loki.ts

```diff
@@ -189,6 +190,7 @@ export class LokiBackend implements DeviceLogsBackend {
 				query: this.getDeviceQuery(ctx),
 				limit: Number.isFinite(count) ? count : 1000,
 				since: '30d',
+				...(start != null ? { start: `${BigInt(start) * 1000000n}` } : {}),
```

Comment:
should this be v ?
```suggestion
				...(typeof start === 'number' ? { start: `${BigInt(start) * 1000000n}` } : {}),
```

Reply:
It's either `undefined` or a number, so both checks should be equivalent?